api/v1alpha1/amaltheasession_types.go

```diff
@@ -65,6 +65,8 @@ type AmaltheaSessionSpec struct {
 
 	// +optional
 	// Additional volumes to include in the statefulset for a session
+	// Volumes used internally by amalthea are all prefixed with 'amalthea-' so as long as you
+	// avoid that naming you will avoid conflicts with the volumes that amalthea generates.
```

Comment:
Should we have a validator that ensure it does not happen ?

Reply:
See #749 and #751